Assets/MixedRealityToolkit.Examples/Experimental/Collections.meta

```diff
@@ -0,0 +1,8 @@
+fileFormatVersion: 2
```

Comment:
Instead of Experimental/Collections, I think Experimental/ScrollingObjectCollection will be much more discoverable for new folks. Even though technically you might want to eventually put stuff under Collections/ScrollingObjectCollection, I would optimize for discoverability with experimental features.

Reply:
I think if all scripts could go under Examples/Experimental/ScrollingObjectCollection, and also SDK/...<the same>, that would be great